### DISCUSSION TITLE
Spoke - Publish Errors

### DISCUSSION THREAD

**Discussion by shinhochoi** [mozilla/hubs#5809] [Hubs Cloud Questions]
AWS Enterprise Service : EC2 C4.Large Model

GLB File size in spoke : 125M
Linux File System Available Size in /dev/xvda1 : 3G (63%)

When publish a scene, Error happens at the moment when uploading file size becomes 75M. 
Errro Publishing Project : Upload failed: Cause: Network Error: Unknown Status. Unknown Error. Possibly a CORS error. 

This error happened about 10 days ago.   We can not  find  why it happens. 
We posted this problem on hubs-cloud discord. No one reply. 

We have to publish our service. we don't have much time. Need help.

![image](https://user-images.githubusercontent.com/9023589/203515082-e71c5d8e-6c75-401d-a5ca-41c6e60af9c5.png)

![image](https://user-images.githubusercontent.com/9023589/203517493-7a1eb86d-6d08-4c80-beaf-8b479899855f.png)

**Comment by matthewbcool** [comment]
If you try to upload through hubs.mozilla.com, do you also see these errors?

General optimization of the scene would be good for performance and might help with the upload. Using texture atlases and reducing the poly count to bring the size down might help.

**Comment by matthewbcool** [comment]
https://discord.com/channels/498741086295031808/819203046931693589/1044356721533001798

update: this is not reproducible on hubs.mozilla.com

**Reply by pattersonbl2** [reply]
An excellent way to test this issue is to SSH into the hubs-cloud instance and run `journalctl  --since "10 min ago" -f `   This will follow the logs produced by your hubs-cloud instance, While the log terminal is up try to publish the scene again to see what errors are produced.  You could also use this guide as well.  lhttps://blog.container-solutions.com/a-guide-to-solving-those-mystifying-cors-issues.

**Comment by shinhochoi** [comment]
After upgrading the current stack to 1.1.3 for enterprise hubs cloud, I can upload 126MB file to spoke and publish scene. There might be a bug in stack.  If  someone got  a same problem, I suggest upgrading stack instead solving in other way.

**Reply by pattersonbl2** [reply]
This seems to be a previous issue, but it was fixed in that 1.1.3 and later templates.  This issue will still be present if your stack is older than 1.1.3.
